src/codeu/chat/client/simplegui/ChatGuiFX.java

```diff
@@ -23,13 +23,32 @@
     private static final double WINDOW_WIDTH = 1000;
     private static final double WINDOW_HEIGHT = 500;
 
+    // login page vars
     private Stage thestage;                         // Holds the scene that user is currently viewing
     private Scene signInScene, mainScene;           // Scenes to hold all the elements for each page
     private Button signInButton;
     private TextField userInput;
     private PasswordField passInput;                // Takes input for username and password
     private ClientContext clientContext;            //
 
+    // main page vars
+    private HBox hboxClient;
+    private HBox hboxInput;
+    private VBox userVBox;
+    private VBox chatVBox;
+    private VBox convosVBox;
+    private BorderPane container;
+    private Button sendButton;
```

Comment:
Are these for a different screen than the login? You might consider splitting this up into two classes: one for the login screen, and another for the main chat window.